Frontend/src/Components/PPC_MPPT/MPPT_CELL.js

```diff
@@ -16,10 +16,9 @@ export default function MPPT_CELL(props){
             bg={props.isCurrMode ? greenBgCol : redBgCol}
             borderColor={borderCol}
             borderWidth='2px'
-            lineHeight='1.6em'
+            lineHeight={props.lineHeight}
```

Comment:
```suggestion
```
It looks like the `lineHeight` prop is never passed to `MPPT_CELL`, so we can remove it.

Reply:
fixed